march_hardware/include/march_hardware/encoder/Encoder.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Project March.
+
+#ifndef MARCH_HARDWARE_ENCODER_H
+#define MARCH_HARDWARE_ENCODER_H
+
+#include <cstddef>
+#include <cstdint>
+
+namespace march
+{
+class Encoder
+{
+public:
+  explicit Encoder(size_t number_of_bits);
+
+  /*
+   * Reads out the encoder from the slave and returns the value in Internal Units (IU).
```

Comment:
It's both actually since this is a base class for those types of encoders. The _the_ refers to the actual encoder instance in use. So this description would be valid for both `absolute_encoder.getAngleIU()` and `incremental_encoder.getAngleIU()`